### PR TITLE
remove incorrect hiddenkeyword for startpage

### PR DESCRIPTION
**Pull request by Findus23**
https://github.com/matomo-org/matomo/pull/16747 only solves half of https://github.com/matomo-org/matomo/issues/16742

for the default to work, the incorrect `hiddenkeyword` has to be removed.

### CODE REVIEW

File: SearchEngines.yml [hunk 1]
```diff
@@ -1355,8 +1355,6 @@ IxQuick:
     params:
       - query
     backlink: 'do/asearch?query={k}'
-    hiddenkeyword:
-      - '/do/asearch'
```
Comment (sgiehl) [inline SearchEngines.yml:1359]: that would actually mean that e.g. `ixquick.com/do/asearch` would not be detected anymore, as only `ixquick.com` and `ixquick.com/` would be allowed for hidden keywords

Comment (Findus23) [reply SearchEngines.yml:1359]: This is intentional, as I can find nothing on startpage.com that uses `/do/asearch` anymore (the links to the website are directly in the `<a>` and I doubt they use it anymore if the referrer policy does the same result but without the disadvantages of a redirect.

Comment (sgiehl) [reply SearchEngines.yml:1359]: Sure. Then it might only make a difference when processing historical data

**Comment by sgiehl** [comment]
closing in favor of #78

**Pull request closed without merging**